src/sections/AddressInfoSection.tsx

```diff
@@ -168,7 +170,7 @@ const TransactionInfoSection = () => {
           </>
         ) : (
           <TableBody>
-            <NoTxMessage>No transactions yet</NoTxMessage>
+            <NoTxMessage>{txListError ?? 'No transactions yet'}</NoTxMessage>
```

Comment:
This will only return `No transactions yet` if `txListError` is specifically `undefined` or `null` ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)). The initial value of `txListError` seems to be `''`. I think what you want here is `txListError || 'No transactions yet'`.